cfgov/preprocessed/js/modules/util/EventObserver.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+/**
+ * EventObserver
+ * @class
+ *
+ * @classdesc Used for creating an object
+ *   that can be used to dispatch and listen to custom events.
+ * @returns {Object} An EventObserver instance.
+ */
+function EventObserver() {
+
```

Comment:
Consider using the native Event constructor for modern browsers and using this for those that don't support it. 

https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Creating_and_triggering_events